libs/icons/readme.md

```diff
@@ -0,0 +1,48 @@
+# Sage Icons
+
+This is the Icon Library for Sage
+
+
+## Figma Export
+In order to export icons you will need to follow the steps listed below.
+
+### Creating a Figma access token
+1. Navigate to [Figma](https://www.figma.com) and login with your credentials.
```

Comment:
```suggestion
1. Navigate to [Figma](https://www.figma.com) and login with your credentials.
```
If you are writing in complete sentences, use punctuation at the end of each numbered list sentence.